setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 100
```

Comment:
Not thrilled about tweaking line length, but I also think that a max of 80 is silly.